panels/power/cc-power-panel.c

```diff
@@ -2368,9 +2369,13 @@ static void
 on_content_size_changed (GtkWidget *widget, GtkAllocation *allocation, gpointer data)
 {
   GtkWidget *box;
+  CcShell *shell;
 
+  shell = cc_panel_get_shell (data);
   box = gtk_widget_get_parent (gtk_widget_get_parent (widget));
-  if (allocation->height < 490)
+
+  if (!shell ||
```

Comment:
`cc_panel_get_shell()` can't return `NULL` as far as I can see, so this can be simplified.